### PR TITLE
Added Uprising rules

### PR DESCRIPTION
**Pull request by seanKenkeremath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,14 +45,18 @@ The player then attempts to catch them hammer after it flips at least once in th
...
 The way that player catches the hammer is how they must perform their strikes.
+If they adjust the grip on their hammer they must drink and immediately end their turn.
+If they attempt to perform a strike (hammer makes contact with the stump or a nail) after regripping they also lose their next turn.
 
 
 Upon catching the hammer, the player must immediately move to the Strike phase without hesitation.
 
 
-If a player fails to catch the hammer, they must drink and immediately end their turn, passing the hammer to the next play in turn order.
+If a player fails to catch the hammer they must drink and immediately end their turn, passing the hammer to the next play in turn order.
```
Comment (willtellis) [inline README.md]: I think having the comma was more correct

File: README.md [hunk 2]
```diff
@@ -45,14 +45,18 @@ The player then attempts to catch them hammer after it flips at least once in th
...
+
+Before flipping the hammer, if the active player is a Lad they may call "Uprising" before flipping the hammer. If they do so then their turn counts as an Uprising (See Uprising section) and Uprising rules apply for the remainder of that turn.
```
Comment (willtellis) [inline README.md]: Commas, get some.

`Before flipping the hammer, if the active player is a Lad, they may call "Uprising" before flipping the hammer. If they do so, then their turn counts as an Uprising (See Uprising section), and Uprising rules apply for the remainder of that turn.`

File: README.md [hunk 3]
```diff
@@ -117,5 +121,17 @@ If the Hammer Test is performed during another player's turn and that player giv
 If a Lord drops their hammer during the Flip phase they are instantly demoted to Lad. Before ending their turn they must drive a new starting nail into the outer ring. 
 
 
+### Uprising
+
+At the start of each game, there is an Uprising counter that begins as 0. A di or some other object may be used to represent this or it can be kept track of mentally by all players as a group.
```
Comment (willtellis) [inline README.md]: "die" is singular

File: README.md [hunk 4]
```diff
@@ -117,5 +121,17 @@ If the Hammer Test is performed during another player's turn and that player giv
...
+
+If the active player is a Lad, they may call "Uprising" before flipping their hammer. If they do then the turn officially counts as an Uprising. The player must perform a double hammer flip followed by one strike without hesitation (note that they do not get two strikes like they normally would). If they hit their nail with that strike then the Uprising was successful and the Uprising counter is incremented by 1. If they drop the hammer, regrip the hammer, or miss their nail the Uprising counter is reset to 0 in which case they must drink and end their turn immediately, passing the hammer to the next player in turn order.
```
Comment (willtellis) [inline README.md]: `If they do, then`

Comment (willtellis) [reply README.md]: `If they hit their nail with that strike, then the Uprising was successful, and the Uprising counter is incremented by 1.`

Comment (willtellis) [reply README.md]: `If they drop the hammer, regrip the hammer, or miss their nail, the Uprising counter is reset to 0, in which case they must drink and end their turn immediately, passing the hammer to the next player in turn order.`

**Review by willtellis — APPROVED** [review]
Commas, get some

File: README.md (hunk 1 above)
Comment (seanKenkeremath) [reply README.md]: Yeah you right. I always second guess my self about the if, then stuff. I'll fix it

**Review by mrketchup — APPROVED** [review]

**Pull request merged**